Sources/RInAppMessaging/Services/MessageMixerService.swift

```diff
@@ -9,6 +9,8 @@ internal enum MessageMixerServiceError: Error {
     case jsonDecodingError(Error)
     case invalidConfiguration
     case tooManyRequestsError
+    case internalServerError(Int)
```

Comment:
Can we use `UInt` instead of `Int`? or the numbers can be negative?

Reply:
Yes, using UInt makes more sense 👍